dataflow/src/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -472,7 +478,7 @@ public boolean equals(Object obj) {
                 return false;
             }
             LocalVariable other = (LocalVariable) obj;
-            return other.element.equals(element);
+            return other.toString().equals(toString());
```

Comment:
Why is the element.equals method insufficient?   Should add a comment to the code.